pkg/endpoint/bpf.go

```diff
@@ -161,7 +161,7 @@ func (e *Endpoint) writeHeaderfile(prefix string, owner Owner) error {
 	fw.WriteString(common.FmtDefineAddress("LXC_MAC", e.LXCMAC))
 	fw.WriteString(common.FmtDefineAddress("LXC_IP", e.IPv6))
 	if e.IPv4 != nil {
-		fmt.Fprintf(fw, "#define LXC_IPV4 %#x\n", binary.BigEndian.Uint32(e.IPv4))
+		fmt.Fprintf(fw, "#define LXC_IPV4 %#x\n", byteorder.HostToNetworkSliceU32(e.IPv4))
```

Comment:
Rebased on master which includes 697b6d18ce17 (bpf: fix drop due to wrong l3/l4 csum in ipv4_redirect_to_host_port, 2017-05-17)